tailwindcss-custom-types.ts

```diff
@@ -1,9 +1,18 @@
+import { NavTWClasses } from '@/components/site/Nav';
+
 type SCDPTailwindTypes =
+  | 'px-gutter-all'
   | 'heading-xl'
   | 'heading-lg'
   | 'heading-md'
   | 'heading-sm'
   | 'heading-xs'
+  | NavTWClasses
+  // @TODO - some transform utilities that are enabled by default
+  // aren't being added to the classname gen. Why?
```

Comment:
Yes. I don't  know why this is. Noticed it in HZTL too. Maybe a bug w/ TW Classnames?

Reply:
Probably. I created a Trello card for one of us to investigate/open up an issue on TW Classnames when we have a little bit of time.

Reply:
https://trello.com/c/4KggUhXG/43-open-issue-for-tailwind-classnames-transform-utilities-not-generated